libqtile/config.py

```diff
@@ -610,3 +640,75 @@ def __repr__(self):
         actions = utils.describe_attributes(self, ['group', 'float',
             'intrusive', 'break_on_match'])
         return '<Rule match=%r actions=(%s)>' % (self.match, actions)
+
+
+class DropDown(configurable.Configurable):
+    """
+    Configure a specified command and its associated window for the SratchPad.
+    That window can be shown in current group and hidden again at keystroke.
+    """
+    defaults = (
+        (
+            'x',
+            0.1,
+            'X position of window as fraction of current screen width. '
+            '0 is the left most position.'
+        ),
+        (
+            'y',
+            0.0,
+            'Y position of window as fraction of current screen height. '
+            '0 is the top most position. To show the window at bottom, '
+            'you have to configure a value < 1 and an appropriate height.'
+        ),
+        (
+            'width',
+            0.8,
+            'Width of window as fraction of current screen width'
+        ),
+        (
+            'height',
+            0.35,
+            'Height of window as fraction of current screen.'
+        ),
+        (
+            'opacity',
+            0.9,
+            'Opacity of window as fraction. Zero is opaque.'
+        ),
+        (
+            'on_focus_lost_hide',
+            True,
+            'Shall the window be hidden if focus is lost? If so, the DropDown '
+            'is hidden if window focus or the group is changed.'
+        ),
+        (
+            'warp_pointer',
+            True,
+            'Shall pointer warp to center of window on activation? '
+            'This has only effect if any of the on_focus_lost_xxx '
+            'configurations is True'
+        ),
+    )
+
+    def __init__(self, name, cmd, **config):
+        """
```

Comment:
You should document the args here.

Reply:
Like for other `configurable.Configurable`s the docmentation is
inherent in `defaults` dict and is contained in generated documentation.

Reply:
As for other `configurable.Configurable`s the (keyword-)args are documented in `defaults` dict and included in generated documentation.